log/aggregator.go

```diff
@@ -0,0 +1,15 @@
+package log
+
+// Aggregator is an interface for pluggable components that collect log messages delivered via
+// some transport mechanism
+type Aggregator interface {
+	// Listen causes an aggregator to begin consuming messages via its underlying transport mechanism
```

Comment:
ah, right - forgot about the mechanics here. maybe this is a better one: "this must _not_ be a blocking function. use it to start the aggregator, and the channel returned by `Stopped()` to wait until the aggregator has stopped listening"